MibTeX/src/de/mibtex/export/Export.java

```diff
@@ -90,7 +97,7 @@ public void checkCrossReferenceResolution(Key key,
         readTags();
     }
 
-    private void extractEntries(BibTeXDatabase database) {
+    private static void extractEntries(BibTeXDatabase database) {
```

Comment:
What is the advantage of changing all methods to static?

Reply:
All these methods are independent from the object's state. So they were basically static all the time without specifying them so explicitly. I suggest that methods that do not read or write an object's state should always be static as their purpose can be misleading otherwise. If these methods would not be static, I would have to invoke them on an object of type `Export` and would expect them to alter this `Export` object. But they don't. Instead, they change static fields in the class `Export` only.